test/test_IN_4.F

```diff
@@ -1,3 +1,11 @@
+C       This is a test for NCEPLIBS-bufr.
+C
+C       This tests a number of library routines and functionality
+C       involved in reading input BUFR messages, as noted in the
```

Comment:
OK, this is not a good comment if it applies to every test.

Put comments that tell what the difference between these programs are, not just that they are tests.

Reply:
First of all, this new top-level comment does clearly state whether the test code is for reading input BUFR messages (IN) or writing output BUFR messages (OUT).  So it's not the same top-level comment for every test code.

Secondly, and as each new top-level comment clearly states, all of the specific routines and functionality covered by each test code are already documented below in the existing comments and print statements for that test code.  Do you want me to just cut+paste that same content a second time into this new top-level comment?